### PR TITLE
Fix abstracts

### PR DESCRIPTION
**Pull request by nzw0301**
Like other my PR, this PR fixes minor typos and uses math syntax.

### CODE REVIEW

File: _posts/2010-03-31-agarwal10a.md [hunk 1]
```diff
@@ -3,12 +3,10 @@ title: Optimal Allocation Strategies for the Dark Pool Problem
...
-  are possible. We extend the previous work on this problem by Ganchev et al (UAI
-  2009) to adversarial scenarios, while also improving over their results in the iid
-  setup. The resulting algorithms are efficient, and are tested on extensive simulations
-  under stochastic and adversarial inputs. Our work also has consequences for other
-  perishable inventory control problems, extending their analyses to adversarial models
-  too.
+  are possible. We extend the previous work on this problem (Ganchev et al., 2009)
+  to adversarial scenarios, while also improving over their results in the iid
+  setup. The resulting algorithms are efficient, and perform well in simulations
+  under stochastic and adversarial inputs.
```
Comment (nzw0301) [inline _posts/2010-03-31-agarwal10a.md:9]: Following the manuscript's abstract
Screenshot 2022-02-11 at 5 03 02


PDF: http://proceedings.mlr.press/v9/agarwal10a/agarwal10a.pdf

File: _posts/2010-03-31-peters10a.md [hunk 2]
```diff
@@ -5,9 +5,9 @@ abstract: Inferring the causal structure of a set of random variables from a fin
   using additive noise models have been suggested to approach the case of continuous
   variables. In many situations, however, the variables of interest are discrete or
   even have only finitely many states. In this work we extend the notion of additive
-  noise models to these cases. Whenever the joint distribution P^(X,Y) admits such
-  a model in one direction, e.g. Y=f(X)+N, N independent of X, it does not admit the
-  reversed model  X=g(Y)+N’, N’ independent of Y as long as the model is chosen in
+  noise models to these cases. Whenever the joint distribution $\mathbf{P}^{(X,Y)}$ admits such
+  a model in one direction, e.g. $Y = f(X)+N$, $N \perp \!\!\! \perp X$, it does not admit the
+  reversed model  $X=g(Y)+\tilde{N}$, $\tilde{N} \perp \!\!\! \perp Y$ as long as the model is chosen in
```
Comment (nzw0301) [inline _posts/2010-03-31-peters10a.md:10]: `\perp \!\!\! \perp` is for the symbol of independence. It shows like this on my Google chrome:

Screenshot 2022-02-11 at 5 10 08

Ref: https://tex.stackexchange.com/a/455701/263149

**Review by lawrennd — APPROVED** [review]
Thanks Kento!

**Comment by nzw0301** [comment]
My pleasure! I appreciate your quick approval!

**Review by lawrennd — APPROVED** [review]
Thanks so much Kento!

**Pull request merged**